ReginaGyimesi/CodeToGive2022_Frontend
# Feature/accessibility-panel

## What does this PR do?

Added a similar panel as SalvaVita has.
Moved voice command inside panel.

## How should it be tested?
Click on accessibility button and then try voice command, it will appear in the bottom right corner. That serves as on and off, and it continously listens until it is swithced off.
